src/Compute/Compute.Test/ScenarioTests/LogAnalyticTests.ps1

```diff
@@ -43,6 +43,67 @@ function Test-ExportLogAnalyticRequestRateByIntervalNegative
         "the given SAS URI";
 }
 
+<#
+.SYNOPSIS
+Test Export Log Analytics group by parameters
+#>
+function Test-ExportLogAnalyticGroupByParameters
+{
+
+    # Setup
+    $rgname = Get-ComputeTestResourceName;
+    $loc = "westus2";
+
+    try
+    {
+        New-AzResourceGroup -Name $rgname -Location $loc -Force;
+        $stoname = 'sto' + $rgname;
+        $stotype = 'Standard_GRS';
+        $container = "contain";
+        New-AzStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype;
+        $key = Get-AzStorageAccountKey -ResourceGroupName $rgname -Name $stoname;
+        Assert-NotNull $key;
+
+        $from = Get-Date -Year 2021 -Month 1 -Day 4 -Hour 9;
+        $to = Get-Date -Year 2021 -Month 1 -Day 5 -Hour 9;
+
+        $context = New-AzStorageContext -StorageAccountName $stoname -StorageAccountKey $key.Value[0];
```

Comment:
Please check the way your module depends on Az.Storage to see whey New-AzStorageContext  cannot be recognized.

Reply:
@msJinLei , this build error is not present on my local VS.

Reply:
Could this error be due to the CredScan check failing instead?